index.html

```diff
@@ -51,13 +51,13 @@ <h2>Run your agents on your machine</h2>
       <div class="LinksInnerBlock">
 
         <h2>Download for your platform</h2>
-        <a href="https://drive.google.com/file/d/1js-XQa1dtkcKlQJsdE2Qq9h2BtIfdHWu" class="OsLink">
+        <a href="https://storage.googleapis.com/fetch-ai-aea-manager/releases/0.6.0/AEA.Manager.Setup.0.6.0.exe" class="OsLink">
           <img src="assets/Windows.svg" alt="Windows" class="OsImg">
         </a>
-        <a href="https://drive.google.com/file/d/1JCDv_4S3I0TROCQL_5OhStQUkubpbV7f" class="OsLink">
+        <a href="https://storage.googleapis.com/fetch-ai-aea-manager/releases/0.6.0/AEA.Manager-0.6.0.dmg" class="OsLink">
           <img src="assets/Macos.png" alt="macOS" class="OsImg">
         </a>
-        <a href="https://drive.google.com/file/d/1HU1uETjFx5rUpV6lIVTOGJSL_8SdIXyD" class="OsLink">
+        <a href="https://storage.googleapis.com/fetch-ai-aea-manager/releases/0.6.0/aea_manager_0.6.0_amd64.deb" class="OsLink">
           <img src="assets/Ubuntu.svg" alt="Ubuntu" class="OsImg">
```

Comment:
Why not link to `latest` and then just change what's there? Means no frontend changes when new versions go live!

Reply:
Then we need to remove version from setup file names. Not that cool.

Reply:
No the files stay as is, just the link needs to resolve to latest.

Reply:
@DavidMinarsch how this can be done? It's a direct link to a file. Name of the folder can be changed but doesn't make much sense as on update we need to rename links anyway.

Reply:
I can ask Ed can it be implemented somehow. But it seems like more needs to be done on their side (redirect to a new version, etc.) Anyway update of links needs to be done, on our side (here) or their (redirect logic).